### PR TITLE
[Java] Adding Armeria 1.26.4

### PR DESCRIPTION
**Pull request by wvuong**

### CODE REVIEW

File: java/armeria/config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+framework:
+  website: https://armeria.dev/
```
Comment (waghanza) [inline java/armeria/config.yaml:2]:
```suggestion
  website: armeria.dev
```

**Review by waghanza — CHANGES_REQUESTED** [review]
Server should listen on `0.0.0.0:3000`

File: java/armeria/pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <packaging>jar</packaging>
+    <name>armeria</name>
+    <url>https://armeria.dev/docs/setup</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>21</java.version>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <armeria.version>1.26.4</armeria.version>
```
Comment (waghanza) [inline java/armeria/pom.xml:18]:
```suggestion
        <armeria.version>[1.26, 1.27)</armeria.version>
```

**Comment by waghanza** [comment]
Hi @trustin @minwoox @ikhoon,

We want to include `armeria` in here. Is this OK for you

-------------

Thanks for your contribution @wvuong.

Could log be totally disabled ?

**Comment by trustin** [comment]
Yes, please. Thank yo for your contribution!

**Comment by wvuong** [comment]
Addressed your code review comments, thanks!

File: java/armeria/src/main/java/benchmark/armeria/BenchmarkApplication.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+public class BenchmarkApplication {
+
+    private static final HttpResponse EMPTY_HTTP_RESPONSE = HttpResponse.of("");
+
+    public static void main(String... args) {
+        Server server = Server.builder()
+                .http(3000)
+                .service("/", (ctx, req) -> EMPTY_HTTP_RESPONSE)
+                .service("/user", (ctx, req) -> EMPTY_HTTP_RESPONSE)
```
Comment (waghanza) [inline java/armeria/src/main/java/benchmark/armeria/BenchmarkApplication.java:14]: should be a post

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
Sorry, CI reveals some errors (500). You should have 3 endpoints :
- GET /
- POST /user
- GET /user/:id (/id is an int)

**Comment by wvuong** [comment]
No problem.  I goofed up the code and misunderstood your POST comment but I just ran the rspec locally and now it passes.

File: java/armeria/pom.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <name>armeria</name>
+    <url>https://armeria.dev/docs/setup</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>21</java.version>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <armeria.version>[1.26, 1.27)</armeria.version>
+        <slf4j.version>1.7.36</slf4j.version>
```
Comment (waghanza) [inline java/armeria/pom.xml:19]: May we use the latest version in maven ? I mean using something dynamic, like + to make sure the build use the latest version available

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
Results available at https://web-frameworks-benchmark.netlify.app/result?f=armeria&l=java

**Pull request merged**